src/main/java/com/jcabi/jdbc/JdbcSession.java

```diff
@@ -240,7 +241,7 @@ public JdbcSession autocommit(final boolean autocommit) {
      *
      * <p>The following types are supported: {@link Boolean},
      * {@link java.sql.Date},
-     * {@link Utc}, {@link Long}, {@link Float}, byte[], {@link Integer}.
+     * {@link Utc}, {@link Long}, {@link Float}, byte[], {@link Integer}, , {@link UUID}.
```

Comment:
It seems there is little a typo, isn't it? Double comma.

Reply:
@HuntMark You are right ! Well seen. :)